auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -191,13 +196,16 @@ import i18n from '@/plugins/i18n'
   }
 })
 export default class EntityManagement extends Mixins(AccountChangeMixin, NextPageMixin) {
+  readonly CommonUtils = CommonUtils
```

Comment:
curious , what is this for ?

Reply:
This is so that code in the template can "see" CommonUtils. It's like a re-export.

Reply:
gotcha..i didn't see you use that in the html.. hm..looks like its passcode/password logic ?may be we should put that logic in one place... i am good other way as well

Reply:
The validation rules are in one place -- the common utils -- but the other rules (such as whether required, or whether one or the other is required (for NRs) belongs in the component, I think).